install/gcp/installer/util/gcloud.py

```diff
@@ -51,6 +52,23 @@ def get_gcloud_info():
             sys.exit(1)
     return project_id, authed_user, is_devshell
 
+def activate_service_account(authed_user, key_path):
+    """Activate the service account with gcloud
+    Args:
+        authed_user (str): service account email
+        key_path (str): absolute path to service account key file
+    """
+
+    return_code, _, err = utils.run_command(
+        ['gcloud', 'auth', 'activate-service-account',
+         authed_user, '--key-file=' + key_path])
```

Comment:
In the context of this method, you should just call the authed_user to be `service_account`.